src/scales/scale.logarithmic.js

```diff
@@ -96,6 +98,12 @@ module.exports = function(Chart) {
 							} else if (value > me.max) {
 								me.max = value;
 							}
+
+                            if(me.minNotZero === null && value !== 0){
```

Comment:
Could be:

    if(value !== 0 && (me.minNotZero == null || value < me.minNotZero)) {
        me.minNotZero = value;
    }

would save some bytes!